homeassistant/components/enigma2/media_player.py

```diff
@@ -226,38 +165,36 @@ async def async_mute_volume(self, mute: bool) -> None:
         """Mute or unmute."""
         await self._device.toggle_mute()
 
-    @property
-    def source(self):
-        """Return the current input source."""
-        return self._device.status.currservice.station
-
-    @property
-    def source_list(self):
-        """List of available input sources."""
-        return self._device.source_list
-
     async def async_select_source(self, source: str) -> None:
         """Select input source."""
         await self._device.zap(self._device.sources[source])
 
     async def async_update(self) -> None:
         """Update state of the media_player."""
         await self._device.update()
-
-    @property
-    def extra_state_attributes(self):
-        """Return device specific state attributes.
-
-        isRecording:        Is the box currently recording.
-        currservice_fulldescription: Full program description.
-        currservice_begin:  is in the format '21:00'.
-        currservice_end:    is in the format '21:00'.
-        """
-        if self._device.status.in_standby:
-            return {}
-        return {
+        self._attr_available = not self._device.is_offline
+        self._attr_extra_state_attributes = {
```

Comment:
Previously the attributes were only set when it was not in standby, now always

Reply:
Oh, sorry :see_no_evil: I am wondering, if it is a good idea at all to remove and create the attributes. (In the long term I want to restructure the attributes anyway, so it doesn't matter for me)

Reply:
They can maybe be migrated to other entities at some point

Reply:
Yes, this is my goal of course. But at first I will finish preparing the config flow and after the config flow is fully implemented, I will take care of these legacies. :slightly_smiling_face: